### PR TITLE
Downward Compatibility

### PR DESCRIPTION
**Pull request by OuYangJinTing**
- For rails5.2, [ActiveRecord::Migrator#initialize](https://github.com/rails/rails/blob/5-2-stable/activerecord/lib/active_record/migration.rb#L1179) only receive three arguments.
- This gem support ruby2.6 in [gemspec file](https://github.com/widefix/actual_db_schema/blob/main/actual_db_schema.gemspec#L19), but numbered parameters is a new feature added in ruby 2.7

_PS: English is not my native language; please excuse typing errors._

### CODE REVIEW

**Comment by OuYangJinTing** [comment]
@ka8725 Can you help me with this PR? Thanks.

File: lib/tasks/db.rake [hunk 1]
```diff
@@ -68,7 +72,7 @@ ActiveRecord::MigrationProxy.prepend(ActualDbSchema::MigrationProxyPatch)
 namespace :db do
   desc "Rollback migrations that were run inside not a merged branch."
   task rollback_branches: :load_config do
-    ActiveRecord::Tasks::DatabaseTasks.raise_for_multi_db(command: "db:rollback_branches")
+    ActiveRecord::Tasks::DatabaseTasks.try(:raise_for_multi_db, command: "db:rollback_branches")
```
Comment (ka8725) [inline lib/tasks/db.rake:75]: Can it check for Rails version as well instead of using `try`? Looks like it was introduced in Rails 6 as well.

Comment (OuYangJinTing) [reply lib/tasks/db.rake]: done

**Review by ka8725 — APPROVED** [review]
Thank you very much for your contribution!

**Pull request merged**